docs/release-notes/cloud/cloud-2023-11-13-jira-slack-webhooks-repo-integrations-removal.md

```diff
@@ -0,0 +1,24 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Removal of Jira, Slack, and Webhooks repository integrations November 13, 2023
+
+On the week of November 13th 2023 we removed the option to create, configure, and delete Jira, Slack, and Webhooks integrations for your repositories. These integrations were available on the repository **Settings**, tab **Integration**.
+
+<!--TODO CY-7215 Add rationale for this removal-->
+
+## Removal of existing integrations
+
+On the week of MM DD YYYY we'll be removing your existing integrations.<!--TODO CY-7215 Update date to remove existing integrations-->
+
+<!--TODO CY-7215 Impacts?-->
```

Comment:
@RaquelHipolito @mrfyda Any impacts to mention, besides that the integrations will stop working?

Reply:
that's it. the integrations and the features they empower